documentation/adr/ADR-0004.md

```diff
@@ -0,0 +1,39 @@
+## Title
+
+Converting the repository into an apt-based application (.deb package)
+
+## Proposal status
+
+What is the status of this decision?
+
+-   Proposed
+
+## Implementation status
+
+-   Work in progress
+
+## Context
+
+Our current application utilizes CURL installation methood and the legacy installation method that is cloning the repository and installing each dependency individually. This requires frequent maintenance by the maintainer and necessitates the manual installation of the dependencies. Switching to packaging, specifically to ".deb" packages, would streamline this process by pre-processing all the dependencies and install the application with all the necessary package dependency. This will eliminate the need for manual intervention by the maintainer unless a new dependency was added/contributed on the open-source repository for each system.
```

Comment:
Feedback on the context of this ADR:

- PT-GUI has three installation methods at the moment; CURL, manual building or pre-built VM images.
- Additionally, containerization of the application has already undertaken development
- Whilst a `.deb` package would be a good additional installation medium, there are some additional challenges that need to be addressed